napari/_qt/layer_controls/qt_vectors_controls.py

```diff
@@ -125,18 +125,18 @@ def __init__(self, layer) -> None:
         )
         self.layer.events.edge_color.connect(self._on_edge_color_change)
 
-    def change_edge_color_property(self, property: str):
+    def change_edge_color_property(self, color: str):
         """Change edge_color_property of vectors on the layer model.
         This property is the property the edge color is mapped to.
 
         Parameters
         ----------
-        property : str
+        color : str
             property to map the edge color to
         """
         mode = self.layer.edge_color_mode
         try:
-            self.layer.edge_color = property
+            self.layer.edge_color = color
```

Comment:
Maybe this should be called `property_name`, it's not really a "color". Or maybe `feature`, since we're going in that direction anyways.

Reply:
so why variable is named `edge_color` not `edge_property`?

Reply:
I suppose it could be called `color_edge_by = property`. Anyway it's based off visualization APIs like Altair and Seaborn, with `visual_channel=property_name` keyword arguments:

```python
alt.Chart(source).mark_circle(size=60).encode(
    x='Horsepower',
    y='Miles_per_Gallon',
    color='Origin',
    tooltip=['Name', 'Origin', 'Horsepower', 'Miles_per_Gallon']
).interactive()
```

https://altair-viz.github.io/gallery/scatter_tooltips.html

I think the alternative is something like `x_coord_from=property_name` etc. Which is a bit verbose.

Reply:
@brisvag (and everyone), I had a brief chat with Andy this morning where we discussed that core devs should be more willing to use Request changes during reviews, because comments are a weak signal to the PR author and also anyone else reviewing, who might see the comment as immaterial to previous reviews. In fact I think in this case it's very important. I think this one merits a "request changes".

Reply:
Yes. Could @brisvag add request changes?

Reply:
Changed. It is ok now?